src/ADAL.PCL/AdalHttpClient.cs

```diff
@@ -103,6 +110,22 @@ public AdalHttpClient(string uri, CallState callState)
                         string[] errorCodes = tokenResponse.ErrorCodes ?? new[] {ex.WebResponse.StatusCode.ToString()};
                         serviceEx = new AdalServiceException(tokenResponse.Error, tokenResponse.ErrorDescription,
                             errorCodes, ex);
+
+                        if (((int) ex.WebResponse.StatusCode).Equals(500) ||
```

Comment:
can we use [https://msdn.microsoft.com/en-us/library/system.net.httpstatuscode.aspx](https://msdn.microsoft.com/en-us/library/system.net.httpstatuscode.aspx)